### PR TITLE
Add German translation (#8)

### PR DESCRIPTION
**Pull request by Yikal**
See #8

### CODE REVIEW

File: Baconit/Strings/de-DE/Resources.resw [hunk 1]
```diff
@@ -0,0 +1,889 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (TiagoDanin) [inline Baconit/Strings/de-DE/Resources.resw]: Remove character: `\ufeff`

File: Baconit/Strings/de-DE/Resources.resw [hunk 2]
```diff
@@ -0,0 +1,889 @@
...
+    <value>Um Kommentare für diesen oder jeden Beitrag zu sehen, platziere deinen Mauszeiger oder Finger auf die Titelleiste und scrolle herunter.</value>
+  </data>
+  <data name="WelcomeText.Text" xml:space="preserve">
+    <value>Wilkommen Quinbd</value>
+  </data>
+  <data name="AccountAgeTB.Text" xml:space="preserve">
+    <value>0 Tage alt</value>
+  </data>
+  <data name="AllTime.Text" xml:space="preserve">
+    <value>Allerzeiten</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/de-DE/Resources.resw]: `allerzeiten` is correct ?

File: Baconit/Strings/de-DE/Resources.resw [hunk 3]
```diff
@@ -0,0 +1,889 @@
...
+    <value>Aktualisiere automatisch den Bildschirmhintergrund</value>
+  </data>
+  <data name="BackgroundInboxCTB.Text" xml:space="preserve">
+    <value>Hintergrund Posteingang wird aktualisiert</value>
+  </data>
+  <data name="BackgroundInboxTB.Text" xml:space="preserve">
+    <value>Hintergrund Posteingang wird aktualisiert</value>
+  </data>
+  <data name="CAllTimeB.Content" xml:space="preserve">
+    <value>Allerzeiten</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/de-DE/Resources.resw]: ^ Same

File: Baconit/Strings/de-DE/Resources.resw [hunk 4]
```diff
@@ -0,0 +1,889 @@
...
+  </data>
+  <data name="CancelCT.Content" xml:space="preserve">
+    <value>Abbrechen</value>
+  </data>
+  <data name="CBestCB.Content" xml:space="preserve">
+    <value>Beste</value>
+  </data>
+  <data name="CBIalltime.Content" xml:space="preserve">
+    <value>allerzeiten</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/de-DE/Resources.resw]: ^ Same

File: Baconit/Strings/de-DE/Resources.resw [hunk 5]
```diff
@@ -0,0 +1,889 @@
...
+    <value>Benutzer</value>
+  </data>
+  <data name="WallITB.Text" xml:space="preserve">
+    <value>Bildschirmhintergrund-Bild Quelle</value>
+  </data>
+  <data name="WebFilter.PlaceholderText" xml:space="preserve">
+    <value>Web-Filter</value>
+  </data>
+  <data name="AllTimeCode.Text" xml:space="preserve">
+    <value>Allerzeiten</value>
```
Comment (TiagoDanin) [inline Baconit/Strings/de-DE/Resources.resw]: ^ Same

File: Baconit/Strings/de-DE/Resources.resw [hunk 6]
```diff
@@ -0,0 +1,889 @@
...
+    <value>Kommentar teilen</value>
+  </data>
+  <data name="SharePostFMI.Text" xml:space="preserve">
+    <value>Beitrag teilen</value>
+  </data>
+  <data name="TapViewEntire.Text" xml:space="preserve">
+    <value>Hier berühren, um die ganze Diskussion anzuzeigen</value>
+  </data>
+  <data name="AboutDescription.Text" xml:space="preserve">
+    <value>Reddunt ist ein wunderschöner Reddit® Client basierend auf Baconit, welche eine Open-Source App ist. Es ist auf allen Windows 10 Geräten verfügbar.
```
Comment (TiagoDanin) [inline Baconit/Strings/de-DE/Resources.resw]: No line-break

**Review by TiagoDanin — CHANGES_REQUESTED** [review]

**Comment by Yikal** [comment]
I changed Allerzeiten to Aller Zeiten. I think both would be correct, but the official site uses Aller Zeiten so I think it would be more appropriate.

**Review by TiagoDanin — APPROVED** [review]
LGTM 💯

**Comment by Spenhouet** [comment]
What keeps this pull request from getting merged?

**Comment by vitordelucca** [comment]
Sorry for the SUPER late merge. Will do it soon! Is just lack of time!

**Pull request merged**